### ISSUE TITLE
Enable automodule to document private methods/variables.

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#197]
I would like to be able to configure automodule to document private methods. This is valuable when documenting abstract classes with private methods.

Ex: Enable full module definition

```
.. automodule::
   :members:
   :private-members:
   :undoc-members:
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/197
- Originally reported by: Anonymous
- Originally created at: 2009-06-12T06:29:02.720

**Comment by shimizukawa** [comment]
_From Daniel Neuhäuser on 2010-04-08 14:12:02+00:00_

Python doesn't have private members do you mean methods prefixed with an underscore?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-08-05 17:44:31+00:00_

This has come up now often enough to be implemented.  Scheduling for 1.1.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-03 22:33:45+00:00_

Now implemented (see #176 and #520).
